src/middlewares/error.middleware.ts

```diff
@@ -33,12 +33,16 @@ const csrfErrorHandler = (err: CsrfError | Error, req: Request, res: Response, n
  * Use this error handler by calling next(e) from within a controller
  * Always keep this as the last handler in the chain for it to work.
  */
-const errorHandler = (err: Error, req: Request, res: Response, _next: NextFunction) => {
+const error = (err: Error, req: Request, res: Response, _next: NextFunction) => {
   logger.errorRequest(req, `An error has occurred. Re-routing to the error screen - ${err.stack}`);
 
   res.status(500).render(config.ERROR_TEMPLATE, {
     templateName: config.ERROR_TEMPLATE
   });
 };
 
-export default [pageNotFound, csrfErrorHandler, errorHandler];
+const errorHandler = [pageNotFound, csrfErrorHandler, error];
```

Comment:
Why is it better to export these like this?

Reply:
it is to be able to export it more easily from the index.ts